src/EnergyPlus/PlantUtilities.cc

```diff
@@ -402,6 +402,12 @@ namespace PlantUtilities {
         // Set flow on node and branch while honoring constraints on actuated node
 
         auto &a_node(DataLoopNode::Node(ActuatedNode));
+        if (LoopNum == 0 || LoopSideNum == 0) {
+            // early in simulation before plant loops are setup and found
+            a_node.MassFlowRate = CompFlow;
+            return;
+        }
```

Comment:
This must have fixed something?

Reply:
And neither should this be here.

Reply:
Oh- right - this fixed an array bounds problem. https://github.com/NREL/EnergyPlus/commit/d136c4c8b92bba4f0f873770ff63c4b81679bfef
@rraustad Thanks for the review and merge.